spring-cloud-dataflow-shell-core/src/test/java/org/springframework/cloud/dataflow/shell/AbstractShellIntegrationTest.java

```diff
@@ -107,39 +106,30 @@ public abstract class AbstractShellIntegrationTest {
 	@BeforeClass
 	public static void startUp() {
 		if (applicationContext == null) {
-			if (System.getProperty(SHUTDOWN_AFTER_RUN) != null) {
-				shutdownAfterRun = Boolean.getBoolean(SHUTDOWN_AFTER_RUN);
-			}
-
-			SpringApplication application = new SpringApplicationBuilder(TestConfig.class).build();
-
-			int randomPort = SocketUtils.findAvailableTcpPort();
+			shutdownAfterRun = Boolean.parseBoolean(System.getProperty(SHUTDOWN_AFTER_RUN, "false"));
```

Comment:
**[UNRELATED]** Cleaning up this method a bit (also replaced `SocketUtils` w/ `TestSocketUtils`)